src/Manager/RuleManager.php

```diff
@@ -34,17 +47,11 @@ class RuleManager extends ManagerBase
     public function query($options = [])
     {
         $url = '/rules';
-        $url .= isset($options['visible_on_page']) ? "&visible_on_page={$options['visible_on_page']}" : '';
-        $url .= isset($options['prefetch']) ? "&prefetch={$options['prefetch']}" : true;
-        $url .= isset($options['sort']) ? "&sort={$options['sort']}" : 'asc';
-        $url .= isset($options['start']) ? "&start={$options['start']}" : 0;
-        $url .= isset($options['rows']) ? "&rows={$options['rows']}" : 10;
-        $url .= isset($options['sort_field']) ? "&sort_field={$options['sort_field']}" : 'updated';
-        $url .= isset($options['status']) ? "&status={$options['status']}" : 'published';
```

Comment:
1) This code is mechanically not working on many levels. It is fixed by use of a new function ```getQueryString()```.

2) Notice the original approach has hardcoded default options. I think it is a bad idea to have SDK specific default options, so I have removed those default and let the queries fallback to Decision API default. If we want to set default, we should do it at Decision API.